onnxscript/ir/serde.py

```diff
@@ -717,6 +758,7 @@ def deserialize_dimension(
     return _core.SymbolicDim(None), denotation
 
 
+@_capture_errors(lambda proto: proto.name)
```

Comment:
Here too: `lambda proto, *_:` ?

Reply:
I did anther pass to fix everything (had more errors). PTAL. Thanks!